src/storage/filestorage.hpp

```diff
@@ -88,7 +88,8 @@ class FileStorage : public aos::NonCopyable {
      * @param data Record to add.
      * @return Error
      */
-    aos::Error Add(const T& data)
+    template <typename F>
```

Comment:
Please add  @param description for filter

Reply:
Done